wrappers/Makefile

```diff
@@ -17,9 +17,9 @@ ifeq ($(REALM_ENABLE_SYNC),1)
 else
 	APPLE_PREPROCESSOR_DEFINITIONS=""
 	WRAPPERS_XCODE_TARGET=wrappers
-	APPLE_BUNDLE_NAME=realm-core-$(CORE_VER).tar.xz
+	APPLE_BUNDLE_NAME= realm-core-cocoa-v$(CORE_VER).tar.xz
 	APPLE_BUNDLE_URL="http://static.realm.io/downloads/core/$(APPLE_BUNDLE_NAME)"
```

Comment:
In OS it appears there's a different calculation logic: https://github.com/realm/realm-object-store/blob/master/CMake/RealmCore.cmake#L196, so not sure which one I should use. Can you link me to script that does the uploading - I wasn't able to find it.